setup.py

```diff
@@ -14,6 +14,9 @@
     for file in os.listdir(datafolder):
         if file.endswith(".zip"):
             extract_archives(os.path.join(datafolder, file), outputdir=tmpdir)
+            
+    for file in os.listdir(datafolder):
+        if file.startswith("INSEE_deces_"):
             preprocess_data(str.split(file, sep=".")[0], inputdir=tmpdir, outputdir=outputdir)
```

Comment:
Fine for me but I don't like a double loop, could we make it in one ?

proposal ?
```
  for file in os.listdir(datafolder):
        if file.endswith(".zip") & file.startswith("INSEE_deces_"):
            extract_archives(os.path.join(datafolder, file), outputdir=tmpdir)
            preprocess_data(str.split(file, sep=".")[0], inputdir=tmpdir, outputdir=outputdir)
```

Reply:
That's the issue, the new zip files (incoming data) don't have the same name pattern so I thought that an extract loop, and a process loop would be a better chance. Also, I'll need to write a slightly different parser for the incoming file since they are not exactly the same...

Reply:
Ok but the your two-loop version will extract all .zip into tmp/ and only the files starting with "INSEE_deces_" will be preprocessed (the other being left alone in the tmp/).

if we want different process for different files, the setup is the place to do it ! 
e.g.


```
# example for multipurpose setup handling different structures
  for file in os.listdir(datafolder):
        if file.endswith(".zip") & file.startswith("INSEE_deces_"):
            # extracts in tmpdir and preprocess 10 years files in outputdir
            extract_archives(os.path.join(datafolder, file), outputdir=tmpdir)
            preprocess_data(str.split(file, sep=".")[0], inputdir=tmpdir, outputdir=outputdir)

        if file.endswith(".zip") & file.startswith("new_file_already_processed_"):
             # only extracts new files in outputdir (no need to preprocess)
            extract_archives(os.path.join(datafolder, file), outputdir=outputdir)

        if file.endswith(".zip") & file.startswith("other_structure_"):
            # extracts in tmpdir and preprocess with other process method in outputdir
            extract_archives(os.path.join(datafolder, file), outputdir=tmpdir)
            preprocess_data2(str.split(file, sep=".")[0], inputdir=tmpdir, outputdir=outputdir)
```

We don't need to add all the cases right now. We just need to be sure that the `INSEE_deces_`files are managed properly. We need to discuss how to manage new files anyway.

Reply:
Ok I agree. I'll update this PR with the multipurpose setup you suggested !